spec.bs

```diff
@@ -447,6 +448,83 @@ To <dfn>build an interest group permissions url</dfn> given a [=origin=] |ownerO
 
 </div>
 
+<h3 id="interest-group-storage-maintenance">Interest Group Storage Maintenance</h3>
+
+There is a job that periodically [=performs storage maintenance=] on the user agent's
+[=interest group set=]. It performs operations such as [=list/removing=] expired or excess
+[=interest groups=]. It's required that [=interest group set=] has no more than:
+  * 2000 [=interest group/owners=];
+  * 1000 [=regular interest groups=] per [=interest group/owner=];
+  * 20000 [=negative interest groups=] per [=interest group/owner=];
+  * <code>10\*1024\*1024</code> total [=interest group/estimated size=] of [=interest groups=] per
+    [=interest group/owner=].
+
+<div algorithm='maintenance'>
+To <dfn>perform storage maintenance</dfn>:
+
+1. Let |ownersAndExpiry| be a new [=ordered map=] whose [=map/keys=] are [=origins=]
+  [=interest group/owner=] and [=map/values=] are [=moments=] [=interest group/expiry=].
+1. [=list/For each=] |ig| of the user agent's [=interest group set=]:
+  1. Let |owner| be |ig|'s [=interest group/owner=].
+  1. If |ownersAndExpiry|[|owner|] [=map/exists=], then [=map/set=] |ownersAndExpiry|[|owner|] to
+    |ig|'s [=interest group/expiry=] if it comes after |ownersAndExpiry|[|owner|].
+  1. Otherwise, [=map/set=] |ownersAndExpiry|[|owner|] to |ig|'s [=interest group/expiry=].
+1. If [=map/size=] of |ownersAndExpiry| |i| &gt; 2000, then set |ownersAndExpiry| to

```

Comment:
I think this line (which has a few adjustments) would be better:

`1. If |ownersAndExpiry|'s [=map/size=] &gt; 2000, then [=map/set=] |ownersAndExpiry| to`

Reply:
ok, you answered this in another PR's [comment](https://github.com/WICG/turtledove/pull/796/files#r1365628511) of preferring `&gt`.